app/views/dashboard/show.html.erb

```diff
@@ -2,22 +2,17 @@
   <div class="govuk-grid-column-full">
     <% if current_user.lead_provider? %>
       <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-xl">
-          <%= @lead_provider.name %>
-        </h1>
-        <div class="index-group">
-          <h2 class="govuk-heading-m">
-<!--            TODO Fix after Ethan code merge for lead providers-->
-            <%= govuk_link_to "Find and add schools", nil %>
-          </h2>
-          <p class="govuk-body">Search for schools you want to recruit and add schools you have partnered with.</p>
-        </div>
-        <div class="index-group">
-          <h2 class="govuk-heading-m">
-            <%= govuk_link_to "Check your schools for #{@lead_provider.cohorts.find_by(start_year: Time.zone.now.year).start_year}", lead_providers_your_schools_path(selected_cohort_id: @lead_provider.cohorts.find_by(start_year: Time.zone.now.year).id) %>
-          </h2>
-          <p class="govuk-body">See which schools have added their early career teachers and mentors.</p>
-        </div>
+        <%= render partial: "navigation" %>
+        <h1 class="govuk-heading-xl"><%= @lead_provider.name %></h1>
+        <h2 class="govuk-heading-m">
+          <!-- TODO Fix after Ethan code merge for lead providers-->
+          <%= govuk_link_to "Confirm your schools", nil %>
+        </h2>
+        <p>Tell the Department for Education (DfE) which schools you have confirmed an agreement with.</p>
+        <h2 class="govuk-heading-m">
+          <%= govuk_link_to "Check your schools", lead_providers_your_schools_path(selected_cohort_id: Cohort.current.id) %>
+        </h2>
```

Comment:
I think the ticket says not to have this?

Reply:
That's right